batchflow/research/results.py

```diff
@@ -78,16 +50,219 @@ class Results:
                             (results.name == 'test_accuracy') & results.layout.isin(['cna', 'can'])]
         ```
     """
-    def __init__(self, path, *args, **kwargs):
+    def __init__(self, path):
         self.path = path
         self.description = self._get_description()
-        self.configs = None
-        self.df = self._load(*args, **kwargs)
+        self.configs = self._load_configs()
+        self.names = list(self.description['executables'].keys())
+        self.variables = [var for unit in self.description['executables'].values() for var in unit['variables']]
 
-    def _get_list(self, value):
-        if not isinstance(value, list):
-            value = [value]
-        return value
+    @property
+    def df(self):
```

Comment:
not quite sure that we'll need this property now. 
It silently starts a very heavy loading and think it is better to make a user explicitly state what they want to load via `load_df` arguments

Reply:
At least we need it for backward compatibility.